.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: clojure
 lein: 2.7.1
+dist: trusty
+sudo: false
```

Comment:
Why do we have both `false` and `required` (below) for `sudo` here?  Would it work to just keep the `sudo` that we already have, `required`?  That appears to be working for our jdk8-only Travis testing in pe-puppet-server-extensions.